lib/dnsimple/client/tlds.rb

```diff
@@ -42,6 +42,38 @@ def all_tlds(options = {})
         paginate(:tlds, options)
       end
       alias :all :all_tlds
+
+      # Gets a TLD's details
+      #
+      # @see https://developer.dnsimple.com/v2/tlds/#get
+      #
+      # @param  tld [#to_s] The TLD name.
+      # @param  options [Hash]
+      # @return [Dnsimple::Response<Dnsimple::Struct::Tld>]
+      #
+      # @raise  [Dnsimple::NotFoundError]
```

Comment:
@jacegu Ideally, yes, we should. All exceptions inherit from `Dnsimple::Error`, and `NotFoundError` inherits from `RequestError`, however they are both raised in different circumstances (e.g the method can raise both a `NotFoundError` and a `RequestError`.